.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Bug report
+about: Create a report to help us improve Zowe Explorer
+labels: bug
+---
+
+<!--
+  Before opening a new issue, please search our existing issues: https://github.com/zowe/vscode-extension-for-zowe/issues
+-->
+
+**Describe the bug**
+
+<!-- A clear and concise description of what the bug is. -->
+
+**To Reproduce**
+
+<!--
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+-->
+
+**Expected behavior**
+
+<!-- A clear and concise description of what you expected to happen. -->
+
+**Screenshots**
+
+<!-- If applicable, add screenshots to help explain your problem. -->
+
+**Desktop (please complete the following information):**
+
+-   OS:
+-   Visual Studio Code Version:
```

Comment:
Can we just say VSCode. There is sometimes confusion between Visual Studio and VSCode. Do we really need the VSCode version more than Zowe Explorer version?

The CLI and plugins version may be useful but if SCS is installed that's possibly more important

Reply:
Done. I've added the change